### ISSUE TITLE
Standards around bridging between `/usr/lib/os-release` and labels/annotations?

### ISSUE THREAD

**Issue by cgwalters** [opencontainers/image-spec#1152]
Has there been any discussion on standard bridging between the [os-release file](https://www.freedesktop.org/software/systemd/man/latest/os-release.html) and standardized labels/annotation names?

I tried searching issues/PRs here and didn't see anything.  This seems like an obvious thing to want.  I think to start at least the `ID` and `ID_LIKE` keys.

**Comment by cgwalters** [comment]
cc @vbatts just because I know you :smile:

**Comment by vbatts** [comment]
nothing official. I've heard some tools pull from os-release for the label/annotations.
It seems like a good and fine pattern. Not sure about mandating such.

**Comment by cgwalters** [comment]
Yeah agree we can't mandate.  But how about adding `org.opencontainers.osrelease.id` as a start which is just the value of `ID` from `/usr/lib/os-release`?

**Comment by vbatts** [comment]
@cgwalters do you have links to places where projects are already doing similar? (maybe not in the `org.opencontainers.*` prefix, but still)

**Comment by cgwalters** [comment]
> @cgwalters do you have links to places where projects are already doing similar? (maybe not in the org.opencontainers.* prefix, but still)

Not currently...but I want to. For example, https://github.com/containers/bootc/issues/807 was recently filed, but actually the systemd side has already "standardized" such a thing:
https://www.freedesktop.org/software/systemd/man/latest/os-release.html#SUPPORT_END=

So if we had a standard like `org.opencontainers.osrelease.*` then we'd automatically have org.opencontainers.osrelease.support-end` which would solve the problem.

**Comment by sudo-bmitch** [comment]
There's been a lot of discussion on #903 on end of support and end of life annotations. One of the bigger points made there is that a commitment to patching listed in the manifest is a bit confusing. You cannot patch an existing image, rather you must create a new one, with a different digest. Extending support, or eliminating it early, cannot be changed in already deployed images since users may have copied images with the old dates. There was also a lot of confusion in that issue where some wanted to support a tag, and not a digest, despite the annotation being part of the manifest which is unaware of the various tags that may reference it.

**Comment by bilal9282kha-ship-it** [comment]
#858 

- #1171 #1292 #1191 #1191 #1225 

-Duplicate of #858

**Comment by bilal9282kha-ship-it** [comment]
[google_privacy_policy_en.pdf](https://github.com/user-attachments/files/24155288/google_privacy_policy_en.pdf)
